src/components/Calculator.js

```diff
@@ -1,45 +1,68 @@
-          <button type="button" className="btn btn-outline-primary">+/-</button>
-          <button type="button" className="btn btn-outline-primary">%</button>
-          <button type="button" className="btn btn-outline-primary right">/</button>
-        </div>
-        <div className="row">
-          <button type="button" className="btn btn-outline-primary">7</button>
-          <button type="button" className="btn btn-outline-primary">8</button>
-          <button type="button" className="btn btn-outline-primary">9</button>
-          <button type="button" className="btn btn-outline-primary right">*</button>
-        </div>
-        <div className="row">
-          <button type="button" className="btn btn-outline-primary">4</button>
-          <button type="button" className="btn btn-outline-primary">5</button>
-          <button type="button" className="btn btn-outline-primary">6</button>
-          <button type="button" className="btn btn-outline-primary right">-</button>
-        </div>
-        <div className="row">
-          <button type="button" className="btn btn-outline-primary">1</button>
-          <button type="button" className="btn btn-outline-primary">2</button>
-          <button type="button" className="btn btn-outline-primary">3</button>
-          <button type="button" className="btn btn-outline-primary right">+</button>
-        </div>
-        <div className="row">
-          <button type="button" className="btn btn-outline-primary zero">0</button>
-          <button type="button" className="btn btn-outline-primary">.</button>
-          <button type="button" className="btn btn-outline-primary right">=</button>
-        </div>
+  const afterClick = (e) => {
+    const buttonName = e.target.innerText;
+    setState(calculate({
+      ...nums,
+    }, buttonName));
+  };
+
+  const { total, operation, next } = nums;
+
+  const result = (
+    <p>
+      {total}
+      {' '}
+      {operation}
+      {' '}
+      {next}
+    </p>
+  );
+
+  return (
+    <div className="Calculator-layout">
+      <div className="cal-input">{result}</div>
+      <div className="row">
+        <button type="button" className="btn btn-outline-primary" onClick={afterClick}>AC</button>
+        <button type="button" className="btn btn-outline-primary" onClick={afterClick}>+/-</button>
+        <button type="button" className="btn btn-outline-primary" onClick={afterClick}>%</button>
+        <button type="button" className="btn btn-outline-primary right" onClick={afterClick}>/</button>
+      </div>
+      <div className="row">
+        <button type="button" className="btn btn-outline-primary" onClick={afterClick}>7</button>
+        <button type="button" className="btn btn-outline-primary" onClick={afterClick}>8</button>
+        <button type="button" className="btn btn-outline-primary" onClick={afterClick}>9</button>
+        <button type="button" className="btn btn-outline-primary right" onClick={afterClick}>*</button>
```

Comment:
- [ ] multiplication operation gives error in the console and does not work fine , kindly fix this
  error :point_down: 
![Screenshot from 2022-06-30 19-34-32](https://user-images.githubusercontent.com/66097371/176752952-b8fe8b0e-a782-4640-ba4c-53f0c7b95b54.png)